docs/source/troubleshooting.rst

```diff
@@ -45,3 +45,9 @@ Removing the volume for RHEL 6 or Centos 6
 Removing the volume for RHEL 7, Centos 7, Fedora 27, or Fedora 28
 
 # docker volume rm qpc-data
+
+Adding Active Directory Credentials
```

Comment:
Let's not reference Active Directory.  There could be other reasons to have a `\`.

How about?
- CLI input with `\`